frontend/src/js/app/RightPane.tsx

```diff
@@ -12,10 +12,11 @@ interface PropsT {
   tabs: TabT[];
 }
 
+// Shrink to size 0 so jQuery/cypress can differentiate between :hidden and :visible
 const Tab = styled("div")<{ isActive: boolean }>`
-  height: 100%;
+  height: ${(isActive) => (isActive ? "100%" : 0 )};
```

Comment:
Cypress uses jQuery under the hood, but i could find a way to filter DOM elements by their style/css directly. The most convienient way seem to use jQueries `:visible/:hidden` selector, which only looks at widths and heights of the elements:
https://api.jquery.com/visible-selector/

Reply:
Do you mean the container that holds the Dropzone and the QueryEditor? We can do that, but from a user handling perspective the visibility "flag" or some thing like it is better. 

I haven't tested it, but i imagine that if i reference the pane by it's id rather than its visibility, it is possible that can d'n'd elements in the QueryEditor eventhough its tab isn't actually in the foreground. This way an error might slip through.

What do you think of that?

Reply:
I think i tried that without success, but basically that is what is done in this video: https://www.youtube.com/watch?v=LxkrhUEE2Qk So I'll try that again, i must have missed something the first time

Reply:
I was able to fix that :) 
The selector is got simpler now